paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModelTest.kt

```diff
@@ -46,7 +47,11 @@ class USBankAccountFormViewModelTest {
             showCheckboxControlledFields = false,
             merchantName = MERCHANT_NAME,
             amount = Amount(5099, "usd"),
-            injectorKey = INJECTOR_KEY
+            injectorKey = INJECTOR_KEY,
```

Comment:
Can you verify this value is set in the input field?

Reply:
sorry this wasn't clear, can you assert that the initial value is the form argument value?

Reply:
Done